history.md

```diff
@@ -0,0 +1,81 @@
+As the first group to introduce hackathons to the student population, FOSSBOX's impact on RIT's culture continues as several hackathons are run every semester.
+FOSSBOX also ran several of their hackathons when they were active, such as [Election Night Hackathon](https://fossrit.github.io/events/2019/11/05/election-night-hackathon/) and Random Hacks of Kindness, as well as several others.
+The original Humanitarian Free and Open-Source Software (HFOSS) seminar class was accredited and students in that class commonly got introduced and involved in the student group as well.
+Remy and FOSSBOX started the [course-to-coop life cycle](https://opensource.com/education/10/3/course-co-op-lifecycle-openinnovationrit) where students in open-source would receive co-op offers from companies that were funding and interested in open-source at RIT. This cycle repeated as those students graduated and got full-time offers and referred those who were in the student club for co-op positions.
+FOSSBOX may have been an informal student group at first but it grew to the point where a full-time staff member was managing FOSSBOX and undergraduate students had part-time paid roles doing the secretarial work for FOSSBOX, both of which were funded by a yearly payment by Red Hat Corporation.
+The FOSSBOX community operated for 9 years, collaborating with organizations in the local area such as [Hacks/Hackers](https://www.hackshackers.com/), [BarCampROC](https://rocwiki.org/BarCamp_Rochester), and the student club [CodeRIT](https://github.com/codeRIT).
+
+
+
+## LibreCorps:
+
+Founded in 2012, LibreCorps' first purpose was as a co-op program after developing educational games under the 'One Laptop Per Child' program.
+LibreCorps serves as an experiential learning program where students are deployed to work on real-world open-source software and data projects, supporting faculty partners with direct software development as well as community management, documentation, design, and other forms of non-code contribution.
+RIT LibreCorps has been partnered intermittently with the UNICEF Office of Innovation since 2014.
+As a successful pilot, LibreCorps reached up to 24 supported faculty-driven software projects!
+
+
+
+## FOSS Minor
+
+Beginning of the Fall 2014 Semester, Rochester Institute of Technology started to offer the first minor in [Free and Open Source Software and Free Culture](https://www.rit.edu/study/free-and-open-source-software-and-free-culture-minor).
+The minor was brought about by additional courses offered by the School of Interactive Games and Media ([IGM](https://www.rit.edu/computing/school-interactive-games-and-media)).
+
+
+
+## FOSS@RIT:
```

Comment:
TBH, I don't think there was ever a "FOSS@RIT" community. It was always FOSS Box or FOSS@MAGIC. FOSS@RIT was just the GitHub organization that everything circled around.

Reply:
news to me haha, i imagine FOSS@RIT started as just a generic center/unchanging name for the rest of this stuff to revolve around?

Reply:
@MoralCode The FOSS Box and FOSS@RIT were equivocally the same thing.